README.md

```diff
@@ -25,10 +25,23 @@ It’s a Rails app that:
 
    ```sh
    $ bundle install
+   ```
+
+5. Create development database 
+
+   ```sh
+   $ createdb webpage-versions-db_development
+   ```
+
+6. Edit the file ~\web-monitoring-db\config\database.yml. Uncomment line 35 and append your postgres database role password	
```

Comment:
The idea with both Figaro and dotenv is that there is a file each person working on the app maintains that is not in version control. For Figaro, it’s `/config/application.yml` and for Dotenv, it’s `/.env`. Users would edit one of those files.